auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -192,6 +196,10 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
         // Facebook sign out
         LoginManager.getInstance().logOut();
 
+        // Twitter sign out
+        if (!Fabric.isInitialized()) TwitterProvider.initialize(activity);
+        Twitter.logOut();
```

Comment:
I think we also want to call:
`Twitter.getSessionManager().clearActiveSession();`